Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedCollectionOperations.swift

```diff
@@ -108,10 +108,10 @@ struct UnifiedBulkWrite: UnifiedOperationProtocol {
         let collection = try context.entities.getEntity(from: object).asCollection()
         let session = try context.entities.resolveSession(id: self.session)
         guard let result = try collection.bulkWrite(self.requests, options: self.options, session: session) else {
-            return .none
+            return .unacknowledgedWrite
         }
         let encodedResult = try BSONEncoder().encode(result)
-        return .bson(.document(encodedResult))
+        return .rootDocument(encodedResult)
```

Comment:
rereading the spec, it turns out a number of operations should have had their results considered root documents. I think I may have missed the "including, but not limited to" part [here](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#allowing-extra-fields-in-root-level-documents). 

a couple of those cases led to failures with the new tests, so I went ahead and updated all of these.